meta-luneos/recipes-multimedia/pulseaudio/pulseaudio-modules-droid-hidl_git.bb

```diff
@@ -18,14 +18,16 @@ SRC_URI = "git://github.com/droidian/pulseaudio-modules-droid-hidl.git;branch=bo
     file://0001-module-hidl-use-PA_MAJORMINOR-as-PA_MODULE_VERSION-.patch \
 "
 
+EXTRA_OECONF = " --with-module-dir=${libdir}/pulseaudio/modules "
```

Comment:
no need to add spaces all around just for case someone doesn't know how to use :append :)

Reply:
Fixed; no idea who you might be talking about ;)